RestApi/Sepes.RestApi/Model/Pod.cs

```diff
@@ -9,12 +10,38 @@ public class Pod
         public ushort id { get; }
         public string name { get; }
         public int studyId { get; }
+        public readonly bool allowAll;
+        public readonly ImmutableList<Rule> incoming;
+        public readonly ImmutableList<Rule> outgoing;
+        public readonly ImmutableList<User> users;
+        public readonly ImmutableList<DataSet> locked;
+        public readonly ImmutableList<DataSet> loaded;
+
+        public Pod(ushort id, string name, int studyId, bool allowAll, IEnumerable<Rule> incoming, IEnumerable<Rule> outgoing, 
+                    IEnumerable<User> users, IEnumerable<DataSet> locked, IEnumerable<DataSet> loaded)
+        {
+            this.id = id;
+            this.name = name;
+            this.studyId = studyId;
+            this.allowAll = allowAll;
+            this.incoming = incoming.ToImmutableList();
+            this.outgoing = outgoing.ToImmutableList();
+            this.users = users.ToImmutableList();
+            this.locked = locked.ToImmutableList();
+            this.loaded = loaded.ToImmutableList();
+        }
 
         public Pod(ushort id, string name, int studyId)
```

Comment:
Look into default argument or calling the big constructor with the small one using public Pod(ushort id, string name, int studyId): this(id, name, studyId, ImmutableList<Rule>.Empty, ....) {} pattern.